index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
Yes, for two reasons:
1. Broberg wanted to keep the site simple, with minimal dependencies on external libs.
2. I like it better that way.
It's easy to get the smooth scrolling back though.